float32 supply_voltage
float32 env_temp
float32 yaw
float32 pitch
float32 roll